src/ascii_dialog/guess.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
```

Comment:
No `main()`. Not a standalone executable.

Reply:
My editor likes to put this into new Python files but yes it doesn't really make sense for this to be here. I'll remove it.